.travis.yml

```diff
@@ -53,11 +51,13 @@ install:
   -
 
 script:
-- GO111MODULE=on TRACE=1 ./golden_test.sh
-- GO111MODULE=on TRACE=1 ./test.sh
-- if [ "$TRAVIS_OS_NAME" = "linux" ]; then GO111MODULE=on TRACE=1 PATH=$PATH:$(pwd) ./test_e2e_v1.sh ; fi
-- if [ "$TRAVIS_OS_NAME" = "linux" ]; then GO111MODULE=on TRACE=1 PATH=$PATH:$(pwd) ./test_e2e_v2.sh ; fi
+# The golden_test_ci.sh check if the the testdata is updated according to the current changes
+# To update the testdata use the Makefile targets `make generate-setup` then `make generate-testdata`
+- ./golden_test_ci.sh
```

Comment:
I'd prefer keeping the names as `golden_test.sh` and `test.sh`, since they are not tied to any CI, but just scripts.

Reply:
I agree. The name test-ci is because of the prow. I will change but, then, it means that the prow will be broken. @alexeldeib could you help us by changing there to merge this one?

Reply:
`test_ci.sh` -> `test.sh` right? and leave `test_e2e.sh` as is?

Reply:
happy to add `golden_test` too, haven't covered that

Reply:
https://github.com/kubernetes/test-infra/pull/15264

Reply:
Now know how to contribute with too. 
We should add in some place here where the prow is implemented too.

Reply:
Let's merge it so @alexeldeib @mengqiy ?
Tracked here the need to doc the tests: https://github.com/kubernetes-sigs/kubebuilder/issues/1194